caddyhttp/fastcgi/fastcgi.go

```diff
@@ -323,6 +326,18 @@ func (h Handler) buildEnv(r *http.Request, rule Rule, fpath string) (map[string]
 	// Some web apps rely on knowing HTTPS or not
 	if r.TLS != nil {
 		env["HTTPS"] = "on"
+		// and pass the protocol details like apache's mod_ssl
+		v, ok := sslProtocolToStringMap[r.TLS.Version]
+		if ok {
+			env["SSL_PROTOCOL"] = v
```

Comment:
Rename to `TLS_PROTOCOL`

Reply:
Although if bajillions of PHP sites expect SSL_PROTOCOL then we can leave it

Reply:
I’m using SSL_PROTOCOL to maintain compatibility with, as you say, the bajillions of sites that use mod_ssl and so expect it.

Reply:
We could implement a mod_ssl compatible mode....

I do not like the use of "SSL", it is outdated

Reply:
I don't either, but let's just use it for now with the env vars because, well, most PHP programs are too ;) (no offense)

Reply:
“SSL” may be outdated, but there is a lot of, for example, PHP code that tests against the SSL_PROTOCOL environment variable (due to mod_ssl)... and if you want to implement something compatible with mod_ssl you have you use what it uses vis-à-vis env vars...

Reply:
Yeah; to clarify, I meant let's use "SSL_" for now, to be compatible, since that is the point of setting the environment variables. But we can/should be able to use TLS everywhere else.